types/google.visualization/index.d.ts

```diff
@@ -67,10 +69,26 @@ declare namespace google {
             setOptions(options: Object): void;
             setView(view_spec: string): void;
         }
-
         //#endregion
+        
+        //#region data
+        export class data {
+            // https://developers.google.com/chart/interactive/docs/reference#group
+            static group(data: DataTable, keys: any[], columns: any[]): DataTable;
+            static group(data: DataView, keys: any[], columns: any[]): DataTable;
+            
+            static sum(values: number[]): number;
+            static avg(values: number[]): number;
+            static min(values: number[]): number;
+            static max(values: number[]): number;
```

Comment:
`values` can have multiple types.

Reply:
These would be called directly by the group() function. An end user invoking them directly would be unlikely. However, being able to know they exist on this class is beneficial. Values generically are allowed to be of any[] because you can build your own custom aggregators, but given these built in aggregators don't make sense taking anything but numbers I think the choice is reasonable.

Reply:
Yes, but from this [table](https://developers.google.com/chart/interactive/docs/reference#provided-aggregation-functions), We can define that `values: number[] | string[] | date[]`

Reply:
And also return type for `min` and `max` can be ` number | string | date`

Reply:
Ok, based on usage in the rest of the class it seemed like unions were being actively avoided. So, I'll correct.